### ISSUE TITLE
Highlighting when Strongs numbers are visible is weird.

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1302]
Do we do anything with this? Maybe not highlight those numbers?

![image](https://user-images.githubusercontent.com/13920678/125226195-f0df9380-e313-11eb-8794-e97f28ac9314.png)

**Comment by tuomas2** [comment]
Agreed. But difficult to fix. Similarly I would like to exclude highlgiht from verse numbers, footnote indicators etc.
